code/modules/mob/status_procs.dm

```diff
@@ -4,6 +4,9 @@
 // eye damage, eye_blind, eye_blurry, druggy, TRAIT_BLIND trait, and TRAIT_NEARSIGHT trait.
 
 
+//hand convulsionies
+/mob/proc/convulsing(amount)
```

Comment:
use status effects what the frick?

Reply:
as in https://github.com/tgstation/tgstation/blob/0b03db27bff11b2276ed0eb879360baa8bb96512/code/datums/status_effects/status_effect.dm

Reply:
reeee why